src/mds/MDSRank.cc

```diff
@@ -2785,6 +2785,7 @@ void MDSRankDispatcher::dump_sessions(const SessionFilter &filter, Formatter *f)
     f->dump_float("uptime", s->get_session_uptime());
     f->dump_int("replay_requests", is_clientreplay() ? s->get_request_count() : 0);
     f->dump_unsigned("completed_requests", s->get_num_completed_requests());
+    f->dump_unsigned("completed_flushes", s->get_num_completed_flushes());
```

Comment:
It seems you based this change on an old master commit. Please rebase.

Reply:
yes. I will close it. thx